sui_core/src/authority/authority_store.rs

```diff
@@ -718,16 +667,6 @@ impl<
             }),
         )?;
 
-        if ALL_OBJ_VER {
```

Comment:
So, does this mean we plan to have full-node and validator to have the same behavior when it comes to storing historic objects?

Reply:
@lxfind the plan was to have pluggable GC behaviour.  So in theory Fullnodes and validators can use different GCs - like validators maybe should have a more aggressive one.